pymodbus/client/base.py

```diff
@@ -118,6 +120,7 @@ def __init__(
         self.params.broadcast_enable = bool(broadcast_enable)
         self.params.reconnect_delay = int(reconnect_delay)
         self.params.reconnect_delay_max = int(reconnect_delay_max)
+        self.params.on_reconnect_callback = on_reconnect_callback
```

Comment:
I miss :parma documentation of the parameter (needed in serial/tcp/udp/tls .py)

Reply:
For consistency, I added the `:param` documentation string in a similar fashion as for the related parameters `reconnect_delay` and `reconnect_delay_max`: these are only documented in [`base.py`](https://github.com/pymodbus-dev/pymodbus/blob/dev/pymodbus/client/base.py#L32-L33) and not in serial/tcp/udp/tls.py .